common/test/acceptance/fixtures/__init__.py

```diff
@@ -1,11 +1,15 @@
 import os
 
 
+HOSTNAME = os.environ.get('BOK_CHOY_HOSTNAME', 'localhost')
+CMS_PORT = os.environ.get('BOK_CHOY_CMS_PORT', '8031')
```

Comment:
I'm either missing something, or I wasn't clear.  Either way, can you clarify for me?

I was stating that '8031' seems like it is hardcoded in places, and that it isn't really an option to change from '8031' and still have this work.  Is that true?  Or, if for some reason I wanted to use '8032' would everything just work?